src/index.html

```diff
@@ -1,17 +1,160 @@
+        </span>
+        <span class="form-item">
+          <label for="birthdate">Full date of birth:
+            <input type="date" name="birthdate">
+          </label>z
+        </span>
+        <span>
+          <label for="term">I accept the term of the agreement
+            <input type="checkbox" name="term">
+          </label>
+        </span>
+      </fieldset>
+      <fieldset class="registration container">
+        <legend>Registration</legend>
+        <span class="form-item">
+          <label for="email">E-mail:
+            <input
+              placeholder="email@example.com"
+              type="email"
+              name="email"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </span>
+        <span>
+          <label for="pass">Password:
+            <input
+            type="password"
+            autocomplete="off"
+            name="password"
+            >
+          </label>
+        </span>
+      </fieldset>
+      <fieldset class="interesting-fact container">
+        <legend>An interesting fact about you!</legend>
+        <span class="form-item">
+          <label for="cats">Do you love cats?
+            <label>
+              <input
+                type="radio"
+                name="cats"
+                value="yes"
+              >
+              yes
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="cats"
+                value="no"
+              >
+              no
+            </label>
+          </label>
+        </span>
+        <span class="form-item">
+          <label for="color">
+            What is your favorite color?
+            <input type="color" name="color">
+          </label>
+        </span>
+        <span class="form-item">
+          <label for="go-to-bed">
+            What time do you go to bed?
+            <input type="time" name="go-to-bed">
+          </label>
+        </span>
+        <span class="form-item">
+          <label for="cars">
+            What are your favorite brand of cars?
+            <select
+              name="cars"
+              id="cars"
+              multiple
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+        </span>
+        <label for="rate-work">
+          How do you rate our work?
+          <input type="range" value="0">
+        </label>
+      </fieldset>
+      <fieldset class="add-info container">
+        <legend>Additional info</legend>
+        <span class="form-item">
+          <label for="comments">Comments:
+            <textarea
+              name="comments"
+              id="commetns"
+              cols="20"
+              rows="2"
+              autocomplete="off"
+              onclick="this.setSelectionRange(0, 0);">
+            </textarea>
```

Comment:
```suggestion
              autocomplete="off"></textarea>
```
you dont need `onclick`, to fix the `textarea` behavior you need just to remove the space between open and closed tags